cache.go

```diff
@@ -82,6 +83,17 @@ func WithExpiration(exp time.Duration) ItemOption {
 	}
 }
 
+// WithKeepTTL is the expiration time to keep existing keys when calling Set. By default, it is replaced by the new time or never expires.
+func WithKeepTTL(_keep ...bool) ItemOption {
```

Comment:
Is this better?
```suggestion
func WithKeepTTL(keep bool) ItemOption {
```

Reply:
Both are feasible, but I prefer to input less code because function names already have a certain intention